### DISCUSSION TITLE
Event type: Conference

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-data-model#50] [Controlled Vocabularies] [answered]
@tammy-culture Which [Artsdata Event type](http://kg.artsdata.ca/resource/ArtsdataEventTypes) should be used for conferences?

Here is an event on SPEC that is a conference - https://console.footlight.io/events/adr:spec-qc-ca_la-richesse-des-arts-autochtones

**Comment by saumier** [comment]
Linked to issue https://github.com/culturecreates/artsdata-data-model/issues/48

**Comment by tammy-culture** [comment]
I had a brief back on forth with Frederic Julien from the Linked Digital Future (LDF) Initiative about this.  There is no appropriate vocabulary for this in ADO nor is there anything appropriate in Schema.  I would like to suggest adding a new ADO category called Conference.

**Reply by saumier** [reply]
@tammy-culture This is the spreadsheet https://docs.google.com/spreadsheets/d/1ud_kVTE8C97ZMoe1uYGQ6rW_dodKqsbqiCutLLHO9jo/edit#gid=0

**Reply by saumier** [reply]
@tammy-culture Why did you call it "Talk Event"?  Isn't "Conference" more common?  And is there a Wikidata event type that is a close match (if yes, please include Wikidata link).

**Reply by tammy-culture** [reply]
@saumier It was a suggestion of Frederic and I am totally fine with Conference.

**Reply by saumier** [reply]
@tammy-culture Is there a Wikidata event type that I can include in your controlled vocabulary as a close match?

**Reply by tammy-culture** [reply]
After a brief back and forth with Frederic Julien, currently there is no exisiting wikidata event type that is a good match.

**Comment by saumier** [comment, marked as answer]
Created [http://kg.artsdata.ca/resource/Conference](http://kg.artsdata.ca/resource/Conference) as an Artsdata Event Type. This URI can be used with an Event's schema:additionalType.
